openmetadata-ui/src/main/resources/ui/src/pages/service/index.tsx

```diff
@@ -642,32 +643,28 @@ const ServicePage: FunctionComponent = () => {
 
     switch (serviceName) {
       case ServiceCategory.DATABASE_SERVICES:
-        return `Deleting this ${service} will also delete ${pluralize(
-          instanceCount,
-          'database',
-          's'
-        )}`;
+        return getEntityDeleteMessage(
+          service || 'Service',
+          pluralize(instanceCount, 'Database', 's')
```

Comment:
The default suffix is 's'.  Can we avoid passing the suffix in all of these cases?

```suggestion
          pluralize(instanceCount, 'Database')
```